docs/sources/installation/mac.md

```diff
@@ -1,116 +1,306 @@
-page_title: Installation on Mac OS X
-page_description: Instructions for installing Docker on OS X using boot2docker.
+page_title: Installation on Mac OS X 
+page_description: Instructions for installing Docker on OS X using boot2docker. 
 page_keywords: Docker, Docker documentation, requirements, boot2docker, VirtualBox, SSH, Linux, OSX, OS X, Mac
 
-# Installing Docker on Mac OS X
+# Install Docker on Mac OS X
 
-> **Note:**
-> Docker is supported on Mac OS X 10.6 "Snow Leopard" or newer.
+Because the Docker daemon uses Linux-specific kernel features, you can't run
+Docker natively in OS X.  Instead, you must install the Boot2Docker application.
+Boot2Docker includes a VirtualBox VM, Docker itself, and the Boot2Docker
+management tool.
 
-Because the Docker Engine uses Linux-specific kernel features, you'll need to use a
-lightweight virtual machine (VM) to run it on OS X. You use the OS X Docker client to
-control the virtualized Docker Engine to build, run, and manage Docker containers.
+The Boot2Docker management tool is a lightweight Linux virtual machine made
+specifically to run the Docker daemon on Mac OS X. The VirtualBox VM runs
+completely from RAM, is a small ~24MB download, and boots in approximately 5s.
 
-To make this process easier, we've built a helper application called
-[Boot2Docker](https://github.com/boot2docker/boot2docker) that installs a
-virtual machine (using VirtualBox) that's all set up to run the Docker daemon.
+Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Boot2Docker.
 
-## Demonstration
 
-<iframe width="640" height="360" src="//www.youtube.com/embed/wQsrKX4588U?rel=0" frameborder="0" allowfullscreen></iframe>
+## Learn the key concepts before installing
+ 
+In a Linux installation, your local machine is the Docker host. The Docker
+client, the Docker daemon, and any containers run directly on this host. This
+means you can address ports on a Docker container using standard addressing
+such as `localhost:8000` or `0.0.0.0:8376`.
```

Comment:
or using the default local UNIX socket `/var/run/docker.sock` (its useful to mention it - as an incorrectly set up OSX docker may mention this socket in its error message.